README.md

```diff
@@ -1,6 +1,6 @@
 ![Tern](/docs/img/tern_logo.png)
 
-[![CircleCI](https://circleci.com/gh/vmware/tern.svg?style=svg)](https://circleci.com/gh/vmware/tern)
+[!CircleCI](https://circleci.com/gh/tern-tools/tern.svg?style=svg)](https://app.circleci.com/pipelines/github/tern-tools/tern)
```

Comment:
Did you mean to remove the `[` character? Looks like there is now an extra `]` bracket now after the `https://circleci.com/gh/tern-tools/tern.svg?style=svg` link.

Reply:
Yup. There's a bracket missing. Let me update.